--- a/etc/apache-maddash.conf	2023-03-21 15:11:15.910735451 -0400
+++ b/etc/apache-maddash.conf	2023-03-21 15:11:16.665476626 -0400
@@ -1,8 +1,8 @@
 ##
 #Setup path to web user interface
 #
-Alias /maddash-webui /usr/lib/maddash/maddash-webui
-<Directory "/usr/lib/maddash/maddash-webui/">
+Alias /maddash-webui /usr/share/maddash/maddash-webui
+<Directory "/usr/share/maddash/maddash-webui/">
     Options +FollowSymLinks +ExecCGI
     DirectoryIndex index.cgi index.html
     AddHandler cgi-script .cgi
