Index: b/etc/apache-maddash.conf
===================================================================
--- a/etc/apache-maddash.conf	2023-12-08 15:50:09.697608714 -0500
+++ b/etc/apache-maddash.conf	2023-12-08 15:50:10.769184262 -0500
@@ -1,8 +1,8 @@
 ##
 #Setup path to web user interface
 #
-Alias /maddash-webui /usr/lib/maddash/maddash-webui
-<Directory "/usr/lib/maddash/maddash-webui/">
+Alias /maddash-webui /usr/share/maddash/maddash-webui
+<Directory "/usr/share/maddash/maddash-webui/">
     Options +FollowSymLinks +ExecCGI
     DirectoryIndex index.cgi index.html
     AddHandler cgi-script .cgi
