--- a/etc/apache-maddash.conf	2023-03-28 17:46:30.406246000 +0200
+++ b/etc/apache-maddash.conf	2023-04-05 19:02:34.000252847 +0200
@@ -1,8 +1,8 @@
 ##
 #Setup path to web user interface
 #
-Alias /maddash-webui /usr/lib/maddash/maddash-webui
-<Directory "/usr/lib/maddash/maddash-webui/">
+Alias /maddash-webui /usr/share/maddash/maddash-webui
+<Directory "/usr/share/maddash/maddash-webui/">
     Options +FollowSymLinks +ExecCGI
     DirectoryIndex index.cgi index.html
     AddHandler cgi-script .cgi
--- a/etc/apache-maddash.conf	2023-04-10 09:23:14.642521043 -0400
+++ b/etc/apache-maddash.conf	2023-04-10 09:23:15.358495773 -0400
@@ -1,8 +1,8 @@
 ##
 #Setup path to web user interface
 #
-Alias /maddash-webui /usr/lib/maddash/maddash-webui
-<Directory "/usr/lib/maddash/maddash-webui/">
+Alias /maddash-webui /usr/share/maddash/maddash-webui
+<Directory "/usr/share/maddash/maddash-webui/">
     Options +FollowSymLinks +ExecCGI
     DirectoryIndex index.cgi index.html
     AddHandler cgi-script .cgi
