Index: b/etc/apache-maddash.conf
===================================================================
--- a/etc/apache-maddash.conf	2023-04-12 15:37:34.549204519 +0200
+++ b/etc/apache-maddash.conf	2023-04-12 15:38:51.822161556 +0200
@@ -1,8 +1,8 @@
 ##
 #Setup path to web user interface
 #
-Alias /maddash-webui /usr/lib/maddash/maddash-webui
-<Directory "/usr/lib/maddash/maddash-webui/">
+Alias /maddash-webui /usr/share/maddash/maddash-webui
+<Directory "/usr/share/maddash/maddash-webui/">
     Options +FollowSymLinks +ExecCGI
     DirectoryIndex index.cgi index.html
     AddHandler cgi-script .cgi
