--- a/etc/apache-maddash.conf	2023-02-28 13:20:43.203119000 +0000
+++ b/etc/apache-maddash.conf	2023-03-28 15:46:30.006322876 +0000
@@ -1,8 +1,8 @@
 ##
 #Setup path to web user interface
 #
-Alias /maddash-webui /usr/lib/maddash/maddash-webui
-<Directory "/usr/lib/maddash/maddash-webui/">
+Alias /maddash-webui /usr/share/maddash/maddash-webui
+<Directory "/usr/share/maddash/maddash-webui/">
     Options +FollowSymLinks +ExecCGI
     DirectoryIndex index.cgi index.html
     AddHandler cgi-script .cgi
