--- a/etc/apache-maddash.conf
+++ b/etc/apache-maddash.conf
@@ -1,9 +1,9 @@
 ##
 #Setup path to web user interface
 #
-Alias /maddash-webui /opt/maddash/maddash-webui 
-<Directory "/opt/maddash/maddash-webui/">
-    Options Indexes +ExecCGI
+Alias /maddash-webui /usr/share/maddash/maddash-webui 
+<Directory "/usr/share/maddash/maddash-webui/">
+    Options Indexes +ExecCGI +FollowSymLinks
     DirectoryIndex index.cgi index.html
     AddHandler cgi-script .cgi
     AllowOverride None
